src/server/object_services/md_store.js

```diff
@@ -181,7 +181,7 @@ class MDStore {
             upload_started: null,
         }, {
             hint: 'null_version_index',
-            sort: { bucket: 1, key: 1, version_enabled: 1 },
+            sort: { bucket: 1, key: 1 },
```

Comment:
there's also `find_object_or_upload_null_version`, that uses a similar query. let's change that too

Reply:
I also subscribed to this approach, however after consideration, I am leaning towards the presented approach. 

The upgrade of sequences is about the client's internal model mapping into the DB and change in its implementation. The current approach makes sense responsibility-wise and makes the code tidier and more readable on both the upgrade script and Postgres client code sides.